<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="abrir">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="abrir_linha">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>811</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_msg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>431</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>929</x>
      <y>11</y>
      <width>131</width>
      <height>602</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="copia">
       <property name="text">
        <string>Copia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="esp_vertic">
       <property name="text">
        <string>Esp. Vertic.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="esp_horiz">
       <property name="text">
        <string>Esp. Horiz.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quantizacao">
       <property name="text">
        <string>Quantiza√ß.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_cores">
       <property name="text">
        <string>Numero de cores</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="num_cor">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>257</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Tons_cinza">
       <property name="text">
        <string>Cinza</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="luminosidade">
       <property name="text">
        <string>Luminosid.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Num para lumin.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="num_luminosidade">
       <property name="minimum">
        <number>-256</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_luminosidade">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="contraste">
       <property name="text">
        <string>Contraste</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Num para contr.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="num_contraste">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_contraste">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Histograma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttom_negative">
       <property name="text">
        <string>Negativo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttom_equalization">
       <property name="text">
        <string>Equalizacao</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_histograma">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>abrir</zorder>
   <zorder>abrir_linha</zorder>
   <zorder>image1</zorder>
   <zorder>image2</zorder>
   <zorder>label_msg</zorder>
   <zorder>label_histograma</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
